src/utils/analytics.ts

```diff
@@ -39,22 +39,28 @@ function getAnonymousId() {
 }
 
 export function init() {
-  if (segment) return; // already initialized
+  if (window['rudderanalytics']) return; // already initialized
 
   getAnonymousId();
 
-  const segmentKey = networkId === 'testnet' ? 'diA7hiO28gGeb9fxn615Xs91uX3GyYhL' : 'gVheHtpTIWpmstSvXjGkSY80nGEXgHX4';
+  const segmentKey = networkId === 'testnet' ? '2R7K9phhzpFzk2zFIq2EFBtJ8BM' : '2R7K9phhzpFzk2zFIq2EFBtJ8BM';
+  const rudderStackOptions = {
+    dataPlaneUrl: "https://nearpavelsqp.dataplane.rudderstack.com",
+  }
 
   const options =
     typeof window === 'undefined'
-      ? {}
+      ? rudderStackOptions
       : {
+          ...rudderStackOptions,
           host: `${window.location.protocol}//${window.location.host}`,
           path: '/api/segment',
         };
 
   try {
-    segment = new Analytics(segmentKey, options);
+    window['rudderanalytics'] = RudderAnalytics;
```

Comment:
```suggestion
    window['rudderanalytics'] = await import("rudder-sdk-js");
```
You will also need to make this function async as a result.

For Next-js projects where code is run server-side(no window object is available), we need to initialize rudderanalytics dynamically like [such](https://github.com/rudderlabs/rudder-analytics-next/commit/4c3c6bb2d50c5c591c855d414832cb7c7fc1d390#diff-a8f92a4b893f8519c26e8fdafb5336b3c26d0812eabf00aad4af39fa693662f8R2)

You can find more details on this issue [here](https://github.com/rudderlabs/rudder-sdk-js/issues/517).